assets/css/style-mobile.css

```diff
@@ -444,6 +446,44 @@ h4 {
   margin: 20px 0;
 }
 
+.web-nav {
+  display: none;
+  background: rgba(96, 112, 255, 0.88);
+  height: 100%;
+  width: 100%;
+  top: 0;
+  margin-left: -15px;
+  position: fixed;
+}
```

Comment:
- [ ] Nice work implementing the mobile menu, it look really good :+1: However the space at the top of the option is too much compared to the Figma design. Kindly reduce it to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-06-21 21-51-07](https://github.com/kambereBr/Portfolio/assets/91142432/5519fff8-9f64-4fbd-b629-e0c4c6d77a2c)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-15 17-26-08](https://github.com/kambereBr/Portfolio/assets/91142432/e1401e94-4b9a-43af-b766-29bb843dc21d)